### ISSUE TITLE
Inline PHP within HTML style attribute causes issues

### ISSUE THREAD

**Issue by czycha** [PrismJS/prism#1112]
Text input:
```php
<div style="<?= $style; ?>">
```

Text output:
```php
<div style="{{{PHP1}}}">
```

HTML output specifically for `{{{PHP1}}}`:
```html
<span class="token attr-value">
  <span class="token punctuation">{</span>
  <span class="token punctuation">{</span>
  <span class="token punctuation">{</span>
  PHP
  <span class="token number">1</span>
  <span class="token punctuation">}</span>
  <span class="token punctuation">}</span>
  <span class="token punctuation">}</span>
</span>
```

If PHP is inline within the style attribute, the text is replaced with some internal token. This does not happen if used within any other type of regular attribute. My guess is that it's tokenizing an internal token (`{{{PHP2}}}`). I don't know anything about how Prism works, so perhaps this guess is not accurate.

**Comment by zeitgeist87** [comment]
This would be fixed with my PR #1100

**Comment by zeitgeist87** [comment]
This was fixed by PR #1100
